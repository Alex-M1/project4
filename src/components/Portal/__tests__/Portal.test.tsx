import React from 'react';
import { createPortal } from 'react-dom';
import renderer from 'react-test-renderer';

jest.mock('rc-util/lib/Portal');

const Drop = () => (
  createPortal(
    <div>hello</div>,
    this.dropContainer,
  )
);

test('Drop renders', () => {
  const component = renderer.create(
    <div>
      <input />
      <Drop />
    </div>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
