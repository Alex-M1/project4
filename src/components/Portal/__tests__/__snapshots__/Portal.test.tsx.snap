// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drop renders 1`] = `
<div>
  <input />
</div>
`;
